core/test/fixtures/fraggle-rock/reports/sample-flow-result.json

```diff
@@ -1798,45 +1798,8 @@
             "displayValue": "",
             "details": {
               "type": "opportunity",
-              "headings": [
-                {
-                  "key": "node",
-                  "valueType": "node",
-                  "label": ""
-                },
-                {
-                  "key": "url",
-                  "valueType": "url",
-                  "label": "URL"
-                },
-                {
-                  "key": "wastedMs",
-                  "valueType": "timespanMs",
-                  "label": "Potential Savings"
-                }
-              ],
-              "items": [
-                {
-                  "node": {
-                    "type": "node",
-                    "lhId": "page-0-IMG",
-                    "path": "1,HTML,1,BODY,0,DIV,0,DIV,4,IMG",
-                    "selector": "body > div#__next > div.flex > img.h-32",
-                    "boundingRect": {
-                      "top": 192,
-                      "bottom": 320,
-                      "left": 38,
-                      "right": 322,
-                      "width": 284,
-                      "height": 128
-                    },
-                    "snippet": "<img src=\"/logo-text.svg\" class=\"h-32 sm:h-44 mt-12 md:mt-24 mb-8 lg:mb-16\" alt=\"Mike's Cereal Shack Logo\">",
-                    "nodeLabel": "Mike's Cereal Shack Logo"
-                  },
-                  "url": "https://www.mikescerealshack.co/logo-text.svg",
-                  "wastedMs": 0
-                }
-              ],
+              "headings": [],
+              "items": [],
```

Comment:
with the `initiatorPath.length` check fix, these are now correctly returning no items because the images are directly in the html